### PR TITLE
update change and create by to record user email

### PR DESCRIPTION
**Pull request by dearrthiti**

### CODE REVIEW

File: flask_appbuilder/models/mixins.py [hunk 1]
```diff
@@ -72,6 +70,21 @@ def get_user_id(cls):
             # log.warning("AuditMixin Get User ID {0}".format(str(e)))
             return None
 
+    @classmethod
+    def get_user_email(cls):
+        try:
+            return g.user.email
+        except Exception as e:
+            # log.warning("AuditMixin Get User ID {0}".format(str(e)))
```
Comment (nutmnop) [inline flask_appbuilder/models/mixins.py:78]: can it permanently remove unused log kub?

File: flask_appbuilder/models/mixins.py [hunk 2]
```diff
@@ -72,6 +70,21 @@ def get_user_id(cls):
...
+            return None
+
+    @classmethod
+    def get_username(cls):
+        try:
+            return g.user.username
+        except Exception as e:
+            # log.warning("AuditMixin Get User ID {0}".format(str(e)))
```
Comment (nutmnop) [inline flask_appbuilder/models/mixins.py:86]: can it permanently remove unused log kub?

File: flask_appbuilder/models/mixins.py [hunk 3]
```diff
@@ -48,17 +48,15 @@ class AuditMixin(object):
...
     @declared_attr
     def created_by_fk(cls):
-        return Column(Integer, ForeignKey('ab_user.id'),
-                      default=cls.get_user_id, nullable=False)
+        return Column(Integer,  default=cls.get_user_email, nullable=False)
 
     @declared_attr
     def created_by(cls):
-        return relationship("User", primaryjoin='%s.created_by_fk == User.id' % cls.__name__, enable_typechecks=False)
+        return relationship("User", primaryjoin='%s.changed_by_fk == User.id' % cls.__name__, enable_typechecks=False)
```
Comment (nutmnop) [inline flask_appbuilder/models/mixins.py:55]: Why it was change from created_by_fk to changed_by_fk in created_by method kub?

Comment (dearrthiti) [reply flask_appbuilder/models/mixins.py:55]: mistake ka, will take it back to created_by_fk

**Comment by nutmnop** [comment]
Look good to me kub :+1:

**Review by nutmnop — APPROVED** [review]

**Pull request merged**